### DISCUSSION TITLE
How to disable the title on the plots?

### DISCUSSION THREAD

**Discussion by LaughingMaann** [colour-science/colour#1004] [General]
<img width="1404" alt="image" src="https://user-images.githubusercontent.com/16868265/176108844-1e786ea6-b8bb-43a5-b596-117ba90147c2.png">

Is there a way to disable the title from showing? I set the illuminants to a custom dictionary, but the long ass title forces the plot to minimize for the sake of title fitting into the plot. Would rather like to keep plot maximal default fig size, but minimize or remove auto-generated title of custome illuminants entirely

**Comment by thomasmansencal** [comment]
Hi @LaughingMaann,

You could pass `title=""` as a kwarg to the plotting definition. Here is some doc on the possible ones: https://colour.readthedocs.io/en/develop/generated/colour.plotting.common.KwargsRender.html#colour.plotting.common.KwargsRender

**Reply by LaughingMaann** [reply]
@KelSolaar  do you pass  **title="" as a kwarg** under annotate_kwargs   ? for example under
Examples
    --------
    >>> annotate_kwargs = [
    ...     {'xytext': (-25, 15), 'arrowprops':{'arrowstyle':'-'}  , **'title':""},**
    ...     {'arrowprops':{'arrowstyle':'-['}},
    ...     {},
    ... ]
    >>> plot_kwargs = [
    ...     {
    ...         'markersize' : 15,
    ...     },
    ...     {   'color': 'r'},
    ...     {},
    ... ]
    >>> plot_planckian_locus_in_chromaticity_diagram(
    ...     ['A', 'B', 'C'],
    ...     annotate_kwargs=annotate_kwargs,
    ...     plot_kwargs=plot_kwargs
    ... )  
    
    using this as exmaple. https://colour.readthedocs.io/en/develop/_modules/colour/plotting/temperature.html 
under
def plot_planckian_locus_in_chromaticity_diagram(

**Reply by LaughingMaann** [reply]
ohhhh my god i love you it's just in the actual function

dictt = {**dictt_red, **dictt_blue}
plot_planckian_locus_in_chromaticity_diagram_CIE1931(
illuminants=dictt,
annotate_kwargs=annotate_kwargs,
plot_kwargs=plot_kwargs, title=""
)

**Reply by thomasmansencal** [reply]
Yes! :)
